eventarc/pubsub/README.md

```diff
@@ -0,0 +1,45 @@
+# Events for Cloud Run – Pub/Sub tutorial
+
+This sample shows how to create a service that processes Pub/Sub events using 
+[the CloudEvents SDK](https://github.com/cloudevents/sdk-python).
+
+## Quickstart
+
+Deploy your Cloud Run service:
+
+```sh
+gcloud builds submit \
+ --tag gcr.io/$(gcloud config get-value project)/eventarc-pubsub
```

Comment:
Should you note that they need to have the Cloud SDK set up for this to work?

Reply:
I'm going to leave details like this for the official docs or for the repo README. Some samples do not even have docs. In the eventarc samples we've left this out.